spanner/tests/system.py

```diff
@@ -258,12 +266,37 @@ def tearDown(self):
         for doomed in self.to_delete:
             doomed.delete()
 
-    def _check_row_data(self, row_data):
-        self.assertEqual(len(row_data), len(self.ROW_DATA))
-        for found, expected in zip(row_data, self.ROW_DATA):
+    def _assert_timestamp(self, value, nano_value):
+        self.assertIsInstance(value, datetime.datetime)
+        self.assertIn(value.tzinfo, (None, UTC))
+        self.assertIs(nano_value.tzinfo, UTC)
+
+        self.assertEqual(value.year, nano_value.year)
+        self.assertEqual(value.month, nano_value.month)
+        self.assertEqual(value.day, nano_value.day)
+        self.assertEqual(value.hour, nano_value.hour)
+        self.assertEqual(value.minute, nano_value.minute)
+        self.assertEqual(value.second, nano_value.second)
+        self.assertEqual(value.microsecond, nano_value.microsecond)
+        if isinstance(value, TimestampWithNanoseconds):
```

Comment:
I want it to allow both `datetime.datetime` and `TimestampWithNanoseconds`. My caller will use both types to exercise the branches [in `_make_value_pb`]1]

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/9870057d5e6d8d58b0ab0e854b03fe0d15543877/spanner/google/cloud/spanner/_helpers.py#L138-L141